backend/server.js

```diff
@@ -0,0 +1,24 @@
+const bodyParser = require("body-parser");
+const express = require("express");
+const app = express();
+
+/* To allow access on headers and also to avoid CORS issues */
+
+app.use(function(req, res, next) {
+    res.header("Access-Control-Allow-Origin", "*");
+    res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
+    next();
+});
```

Comment:
we can expand on this a little like so `app.use((req, res, next) => {
  res.header('Access-Control-Allow-Origin', '*');
  res.header('Access-Control-Allow-Headers', 'Origin, X-Requested-With, Content-Type, Accept, Authorization');
  if (req.method === 'OPTIONS') {
    res.header('Access-Control-Allow-Methods', 'GET, PUT, POST, PATCH, DELETE');
    return res.status(200).json({});
  }
  next();
});`
and move it to app.js